src/transformers/models/tvlt/feature_extraction_tvlt.py

```diff
@@ -0,0 +1,521 @@
+# coding=utf-8
```

Comment:
My first comment is that we usually use a `ImageProcessor` class to wrap the image processing, while FeatureExtractor is kept for audio. Let's try to follow that too!